### PR TITLE
HIP 113: Reward CBRS As Experimental

### PR DESCRIPTION
**Pull request by rawrmaan**
Adds a new proposal to adjust PoC rewards for CBRS radios to reflect their limited utility and experimental status on the network.

Rendered view: https://github.com/rawrmaan/HIP/blob/cbrs-experimental/xxxx-reward-cbrs-as-experimental.md

### CODE REVIEW

**Comment by TysonsTech** [comment]
This is the way.

**Comment by riobah** [comment]
there is no reward weight based on the cbrs radio type, like "high power outdoor" AFAIK. 
So the "Changes to Templated Coverage Points" should be removed, right? the "examples" section is still valid without this "Changes to Templated Coverage Points" section.

**Comment by riobah** [comment]
I believe the model predicting the change is pretty accurate. 
We have exported the boosted coverage points for the all network considering overlapping coverage. 
CBRS share of the coverage points go to 29.40% from 80.53%

Data and the analysis can be found at this sheet: 
https://docs.google.com/spreadsheets/d/1MXXq0AxOj-kV6SlGJN2dLwAhR0eYCGvRDwMDTEABE9I
It allows you to change the proposed parameters and see the result too

**Comment by rawrmaan** [comment]
> there is no reward weight based on the cbrs radio type, like "high power outdoor" AFAIK. So the "Changes to Templated Coverage Points" should be removed, right? the "examples" section is still valid without this "Changes to Templated Coverage Points" section.

That's a good question, and I'm not sure of the answer. I believe I heard someone from Nova mention that Templated Rewards were being used in Mexico due to missing obstruction data, but I don't know if that is still the case, or ever was really. Maybe @madninja could chime in.

**Comment by rawrmaan** [comment]
Upon further investigation, it seems like "Templated Coverage Map" for coverage in MX became an overloaded term starting in HIP 93.

The new term refers to a static template for outdoor radios in absence of obstruction data.

The old term referred to the now-obsolete rewards multipliers that exist before Modeled Coverage, so I've removed the Templated Coverage changes from this HIP.

File: xxxx-reward-cbrs-as-experimental.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+- Category: Economic
+- Original HIP PR: XXX
+- Tracking Issue: XXX
+- Vote Requirements: veMOBILE Holders
+
+## Summary
+
+This Helium Improvement Proposal (HIP) suggests adjusting Proof-of-Coverage (PoC) rewards for CBRS radios to reflect their limited utility and experimental status on the network.
+
+## Related Prior HIPs
```
Comment (abhay) [inline xxxx-reward-cbrs-as-experimental.md:14]:
```suggestion

**Disclosure**: Please note that this HIP is written by a Board Member of the [Helium Foundation](https://helium.foundation) solely in a personal capacity. The Foundation was not involved in the drafting of the proposal and has taken no formal position for or against the HIP as of the date of drafting.

## Related Prior HIPs
```

Comment (abhay) [reply xxxx-reward-cbrs-as-experimental.md:14]: Hope you're good with this addition @rawrmaan.

File: xxxx-reward-cbrs-as-experimental.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+
+- [HIP 74](0074-mobile-poc-modeled-coverage-rewards.md) established modeled coverage for the MOBILE subDAO.
+
+- [HIP 85](0085-mobile-hex-coverage-limit.md) changed the limit of outdoor radios eligible for PoC rewards from 5 to 3, and introduced ranking multiplier.
+
+- [HIP 93](0093-addition-of-wifi-aps-to-mobile-subdao.md) introduced the addition of Wi-Fi Access Points and certain limitations.
+  
+- [HIP 101](0101-equalizing-poc-rewards-across-wifi-and-cbrs.md) proposed increasing Wi-Fi coverage points by 1.5x (indoor) and 2.5x (outdoor), but failed to pass a vote.
+
```
Comment (abhay) [inline xxxx-reward-cbrs-as-experimental.md:23]:
```suggestion

**Disclosure**: Please note that this HIP is written by a Board Member of the [Helium Foundation](https://helium.foundation) solely in a personal capacity. The Foundation was not involved in the drafting of the proposal and has taken no formal position for or against the HIP as of the date of drafting.

```
Hope you're good with this addition, @rawrmaan

**Comment by riobah** [comment]
updated the data for Feb29 epoch: https://docs.google.com/spreadsheets/d/1R6y9R2B9Ljan4Spqf0k33ubaufbRTgX_k2FHNJAZhww

**Comment by abhay** [comment]
Lots of quick updates here. @rawrmaan do you believe this is ready to land for a full discussion?

**Comment by rawrmaan** [comment]
I believe so. I don't have any outstanding feedback points from anyone that I'm aware of.

**Pull request merged**